micro-benchmarks/nccl-tests/slurm/topology-aware-nccl-tests/submit_nccl_test_container.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+# Script to submit comprehensive NCCL tests with 2 and 4 nodes
+# Tests all collective operations with different split masks
+
+set -e
+
+# Configuration
+NODE_COUNTS=(2 4 8 16)
+APPS_PATH="/fsxl"
+TEST_TYPES=("allreduce" "allgather" "reducescatter" "alltoall")
+# TEST_TYPES=("allreduce" )
+SPLIT_MASK=("0x0" "0x7")
+# SPLIT_MASK=("0x0")
+ENABLE_NCCL_DEBUG="false"
+
+# Colors for output
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+echo -e "${GREEN}Starting comprehensive NCCL test submission...${NC}"
+echo -e "${BLUE}Configuration:${NC}"
+echo "  Node counts: ${NODE_COUNTS[*]}"
+echo "  Apps path: $APPS_PATH"
+echo "  Test types: ${TEST_TYPES[*]}"
+echo "  split masks: ${SPLIT_MASK[*]}"
+echo "  NCCL Debug: $ENABLE_NCCL_DEBUG"
+echo ""
+
+# Counter for submitted jobs
+job_count=0
+submitted_jobs=()
+
+# Create job tracking files
+timestamp=$(date +"%Y%m%d_%H%M%S")
+job_ids_file="logs/submitted_jobs_${timestamp}.txt"
+job_details_file="logs/job_details_${timestamp}.csv"
+
```

Comment:
```suggestion
mkdir -p logs
```
Otherwise we get `./submit_nccl_test_container.sh: line 43: logs/job_details_20250924_070312.csv: No such file or directory`.